dockerfiles/Caffe2OnnxDockerfile

```diff
@@ -8,10 +8,20 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
        liblmdb-dev libopencv-dev libsnappy-dev libprotobuf-dev \
        protobuf-compiler python-dev libgflags-dev && pip install future==0.16.* protobuf==3.5.*
 
-RUN git clone --recursive https://github.com/caffe2/caffe2.git && cd caffe2 \
-    && git checkout 409e96d9ebd9963e10fd1eaa0a02cfbb8623650e \
-    && git submodule update --init && mkdir build && cd build && cmake .. -DUSE_MPI=OFF \
-    && make -j8 install
+
+RUN git clone --recursive https://github.com/pytorch/pytorch.git \
+	&& cd pytorch \
+	&& git submodule update --init \
+	&& git checkout d695624efe618fbb0c267d564c3a7dabefd32114 \
+	&& mkdir build && cd build && cmake .. -DUSE_MPI=OFF \
+	&& make -j8 install
+
+
+#RUN git clone --recursive https://github.com/caffe2/caffe2.git && cd caffe2 \
```

Comment:
You probably don't want to comment this out, as the Docker container won't work without it. We just want to not build the container in `build_docker_images.sh`. That way when we can add back support, we can just uncomment the line in that script.